agent/cmd/root.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cliExitInterceptor func(code int)
+)
+
+var rootCmd = cobra.Command{
+	Use: "agent",
+	Short: "Manages the tracetest agent"
+	Long: "Manages the tracetest agent"
+}
```

Comment:
In the tracetest CLI we are going to import this and add it as a child of the `tracetest` command. This way you can call it via `tracetest agent start`